extensions/wikia/EditPageLayout/js/plugins/Widescreen.js

```diff
@@ -91,15 +90,15 @@
 		},
 
 		toggle: function() {
-			this.setState(!this.getState());
+					this.setState(!this.getState());
 			this.saveState();
 
 			// toolbar height can change - resize the editor (BugId:5694)
 			this.modeChanged();
 		},
 
 		loadState: function( initial ) {
-			var wide = $.storage.get(this.storageEntry) == true;
+					var wide = $.storage.get(this.storageEntry) == true;
 			this.editor.log('widescreen::load() - wide = '+(wide?'true':'false'));
 			if (initial) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)

[project-id]: wikia:app